using_images/other_images/jenkins.adoc

```diff
@@ -75,8 +155,16 @@ The Jenkins password can be configured with the following environment variable:
 |Variable name |Description
 
 |`*JENKINS_PASSWORD*`
-|Password for the `*admin*` user.
+|Password for the `*admin*` user when using default Jenkins authentication.
+
+ifdef::openshift-origin[]
+|`*OPENSHIFT_ENABLE_OAUTH*`
+|Determines whether the OpenShift Login plugin manages authentication when logging into Jenkins.
 
+|`*OPENSHIFT_PERMISSIONS_POLL_INTERVAL*`
+|Specifies in milliseconds how often the OpenShift Login plugin polls {product-title} for the permissions associated with each user defined in Jenkins.
```

Comment:
i guess we didn't learn our lesson about defaulting to milliseconds on this stuff?  :)

can we change this to seconds please?